READMEs/simple-remote.md

```diff
@@ -34,6 +34,11 @@ export REMOTE_TEST_PRIVATE_KEY1=<your REMOTE_TEST_PRIVATE_KEY1>
 export REMOTE_TEST_PRIVATE_KEY2=<your REMOTE_TEST_PRIVATE_KEY2>
 ```
 
+To avoid using Infura keys, especially for Polygon networks, use this command in the console:
```

Comment:
For polygon-main, I don't use the `maticcivil` RPC because it changed to `https://polygon-rpc.com`. I changed into
"""
The brownie default RPCs require you to have your own infura account, and corresponding token WEB3_INFURA_PROJECT_ID.

    If you have an infura account: set the envvar WEB3_INFURA_PROJECT_ID
    If not: one way is to get an infura account. Simpler yet is you can bypass the need for it, by changing to RPCs that don't need tokens. The command below replaces infura RPCs with public RPCs:

"""